KITE-Extras/src/main/java/io/cosmosoftware/kite/instrumentation/Networks.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) CoSMo Software Consulting Pte. Ltd. - All Rights Reserved
+ */
+
+package io.cosmosoftware.kite.instrumentation;
+
+import io.cosmosoftware.kite.exception.KiteTestException;
+
+import javax.json.JsonArray;
+import java.util.HashMap;
+
+/**
+ * The type Networks.
+ */
+public class Networks extends HashMap<String, Network> {
```

Comment:
rename to NetworkProfile

I think @sajidcosmo need to make this an OGM entity to match with the model NetworkProfile